sites/public/src/components/browse/FilterDrawerHelpers.tsx

```diff
@@ -0,0 +1,351 @@
+            type="currency"
+            prepend="$"
+            register={props.register}
+            getValues={props.getValues}
+            setValue={props.setValue}
+            defaultValue={props.filterState?.[ListingFilterKeys.monthlyRent]?.minRent}
+          />
+        </Grid.Cell>
+        <Grid.Cell>
+          <Field
+            id={`${ListingFilterKeys.monthlyRent}.maxRent`}
+            name={`${ListingFilterKeys.monthlyRent}.maxRent`}
+            label={t("listings.maxRent")}
+            type="currency"
+            prepend="$"
+            register={props.register}
+            getValues={props.getValues}
+            setValue={props.setValue}
+            defaultValue={props.filterState?.[ListingFilterKeys.monthlyRent]?.maxRent}
+          />
+        </Grid.Cell>
+      </Grid.Row>
+      <Grid.Row>
+        <Grid.Cell>
+          <Field
+            id={ListingFilterKeys.section8Acceptance}
+            name={ListingFilterKeys.section8Acceptance}
+            label={t("listings.section8Acceptance")}
+            labelClassName={styles["filter-checkbox-label"]}
+            type="checkbox"
+            register={props.register}
+            inputProps={{
+              defaultChecked: isTrue(props.filterState?.[ListingFilterKeys.section8Acceptance]),
+            }}
+          />
+        </Grid.Cell>
+      </Grid.Row>
+    </Grid>
+  </fieldset>
+)
+
+// built to support filtering by url decoding and filtering by state directly from form data
+export const encodeFilterDataToBackendFilters = (data: FilterData = {}): ListingFilterParams[] => {
+  const filters: ListingFilterParams[] = []
+  Object.entries(data).forEach(([filterType, userSelections]) => {
+    // individual filters not yet implemented
+    if (individualFilters.includes(ListingFilterKeys[filterType])) {
+      Object.entries(userSelections).forEach((field) => {
+        if (field[1]) {
+          const filter = {
+            $comparison: EnumListingFilterParamsComparison["="],
+          }
+          filter[filterType] = field[0]
+          filters.push(filter)
+        }
+      })
+    } else if (arrayFilters.includes(ListingFilterKeys[filterType])) {
+      const selectedFields = []
+      Object.entries(userSelections).forEach((field) => {
+        if (field[1]) {
+          if (filterType === ListingFilterKeys.bedroomTypes) {
+            selectedFields.push(unitTypeMapping[field[0]]?.value)
+          } else {
+            selectedFields.push(field[0])
+          }
+        }
+      })
+      if (selectedFields.length > 0) {
+        const filter = {
+          $comparison: EnumListingFilterParamsComparison["IN"],
+        }
+        filter[filterType] = selectedFields
+        filters.push(filter)
+      }
+    } else if (
+      booleanFilters.includes(ListingFilterKeys[filterType]) &&
+      // filter data direct from form is boolean, decoded from url is string
+      (userSelections == true || userSelections === "true")
```

Comment:
nit: `userSelections === true`

Or you could do `[true, "true"].includes(userSelections)` but that might be less readable